ydb/docs/en/core/yql/reference/syntax/delete.md

```diff
@@ -37,3 +37,18 @@ $to_delete = (
 DELETE FROM my_table ON
 SELECT * FROM $to_delete;
 ```
+
+## BATCH DELETE {#batch-delete}
+
+Deletes data from large tables that cannot be deleted using the standard `DELETE` method due to transaction limits and the risk of potential transaction locks invalidation failure. It independently applies deletions to each partition of the specified table, removing a limited number of rows per iteration (by default, 10000 rows). The query is executed in a non-transactional mode. In case of an error, changes are not rolled back. The semantics are inherited from the standard `DELETE` with the following restrictions:
```

Comment:
Move the "non-transactional" part to the beginning, along the lines of: "`BATCH DELETE` allows to non-transactionally delete a large number of rows from tables to circumvent transaction limits and risk of transaction lock invalidation failures..."